beast/examples/phat_small/datamodel.py

```diff
@@ -289,6 +287,5 @@ def get_obscat(obsfile=obsfile, distanceModulus=distanceModulus,
     obs: GenFluxCatalog
         observation catalog
     """
-    obs = GenFluxCatalog(obsfile, distanceModulus=distanceModulus,
-                         filters=filters)
+    obs = GenFluxCatalog(obsfile, filters=filters)
```

Comment:
Wait -- but instead of placing observed data at 10 pc, the current convention was to place both the observations and models at the distance of the galaxy.  We need to keep distanceModulus in order to convert observations from observed fluxes to intrinsic fluxes.  See my other comment on obervationmodel/observations.py.